src/screens/Pagination.js

```diff
@@ -71,6 +71,22 @@ const PaginationPage = () => (
           <GenericMargin />
         </Property>
 
+        <Property name="messages">
+          <Description>
+            Custom messages for Pagination. Used for accessibility by screen
+            readers.
```

Comment:
Mike made a good point about this https://github.com/grommet/grommet-site/pull/541/files#r2060622200. Maybe we could change the description to something like this:
```suggestion
            Custom messages for Pagination. Used for internationalization and customization of default strings and accessibility announcements.
```

Reply:
should we change all of them then? 
```
Custom messages for Pagination. Used for accessibility by screen
            readers.
``` 

I just grab this messaging that we were already using in other components

Reply:
Yeah I saw we have this other places. Don't worry about changing it everywhere for this PR. I think just change the ones that we are adding in the PR

Reply:
I will just change messaging description overall so we stay consistent